### ISSUE TITLE
 Invalid Input Error

### ISSUE THREAD

**Issue by betsykunjuz** [embedchain/embedchain#104]
Traceback below:

```
"exception": "duckdb.InvalidInputException: Invalid Input Error: File 'db/chroma-embeddings.parquet' too small to be a Parquet file",
"exc": "["Traceback (most recent call last):
File \"apps/frappe/frappe/app.py\", line 66, in application
response = frappe.api.handle()
File \"apps/frappe/frappe/api.py\", line 54, in handle
return frappe.handler.handle()
File \"apps/frappe/frappe/handler.py\", line 45, in handle
data = execute_cmd(cmd)
File \"apps/frappe/frappe/handler.py\", line 83, in execute_cmd
return frappe.call(method, **frappe.form_dict)
File \"apps/frappe/frappe/init.py\", line 1607, in call
return fn(*args, **newargs)
File \"apps/chatbot/chatbot/api.py\", line 26, in get_data
naval_chat_bot = App()
File \"env/lib/python3.10/site-packages/embedchain/embedchain.py\", line 37, in init
db = ChromaDB()
File \"env/lib/python3.10/site-packages/embedchain/vectordb/chroma_db.py\", line 22, in init
super().init()
File \"env/lib/python3.10/site-packages/embedchain/vectordb/base_vector_db.py\", line 3, in init
self.client = self._get_or_create_db()
File \"env/lib/python3.10/site-packages/embedchain/vectordb/chroma_db.py\", line 25, in _get_or_create_db
return chromadb.Client(self.client_settings)
File \"env/lib/python3.10/site-packages/chromadb/init.py\", line 31, in Client
api = system.instance(API)
File \"env/lib/python3.10/site-packages/chromadb/config.py\", line 160, in instance
impl = type(self)
File \"env/lib/python3.10/site-packages/chromadb/api/local.py\", line 61, in init
self._db = self.require(DB)
File \"env/lib/python3.10/site-packages/chromadb/config.py\", line 106, in require
inst = self._system.instance(type)
File \"env/lib/python3.10/site-packages/chromadb/config.py\", line 160, in instance
impl = type(self)
File \"env/lib/python3.10/site-packages/chromadb/db/duckdb.py\", line 445, in init
self.load()
File \"env/lib/python3.10/site-packages/chromadb/db/duckdb.py\", line 503, in load
self._conn.execute(\nduckdb.InvalidInputException: Invalid Input Error: File 'db/chroma-embeddings.parquet' too small to be a Parquet file\n"]"
```

**Comment by taranjeet** [comment]
This issue is not happening for me. Closing for now.
Please feel free to reopen if the issue still occurs.

**Comment by crojascreangel** [comment]
I'm struggling with the same issue. how do you fix it?

**Comment by crojascreangel** [comment]
I deleted the folder and then rebuilt this, but this is not a practical way. I need that data to persist and they being queriable.

**Comment by cachho** [comment]
> I deleted the folder and then rebuilt this, but this is not a practical way. I need that data to persist and they being queriable.

It should be that way and I haven't had that happen since the first week embedchain came out.

The way I solved it back then is by using ChromaDB in client/server mode. Then your data is independent from the methods consuming it.

Could it be that you created the database a long time ago? ChromaDB has since been updated to use Sqlite 3. That might've made an old database incompatible
